riscv/src/large_field/code_gen.rs

```diff
@@ -296,6 +296,24 @@ fn preamble(field: KnownField, runtime: &Runtime, with_bootloader: bool) -> Stri
     col witness tmp3_col;
     col witness tmp4_col;
 
+    // ================ Publics ==================
+    std::machines::write_once_memory::WriteOnceMemory publics;
+    instr commit_public X, Y link => publics.access(tmp1_col, tmp2_col)
+        link ~> tmp1_col = regs.mload(X, STEP)
+        link ~> tmp2_col = regs.mload(Y, STEP);
+
+    /*
+    public hash_0 = main_publics::value(0);
+    public hash_1 = main_publics::value(1);
+    public hash_2 = main_publics::value(2);
+    public hash_3 = main_publics::value(3);
+    public hash_4 = main_publics::value(4);
+    public hash_5 = main_publics::value(5);
+    public hash_6 = main_publics::value(6);
+    public hash_7 = main_publics::value(7);
+    */
```

Comment:
Commented out to get the tests to run. When uncommented we get this:

```console
thread 'main' panicked at pil-analyzer/src/expression_processor.rs:374:18:
TODO: Handle this up in the code: "Value symbol not found: main_publics::value"
```

We need to decide if we want to go ahead without these or fix first.